src/theory/quantifiers/sygus/ce_guided_single_inv.cpp

```diff
@@ -394,32 +391,26 @@ Node CegSingleInv::reconstructToSyntax(Node s,
     }
   }
 
-  // debug solution
-  if (!d_sol->debugSolution(d_solution))
+  if (Trace.isOn("cegqi-stats"))
   {
-    // This can happen if we encountered free variables in either the
-    // instantiation terms, or in the instantiation lemmas after postprocessing.
-    // In this case, we fail, since the solution is not valid.
-    Trace("csi-sol") << "FAIL : solution " << d_solution
-                     << " contains free constants." << std::endl;
-    Warning() <<
-        "Cannot get synth function: free constants encountered in synthesis "
-        "solution.";
-    reconstructed = -1;
-  }
-  if( Trace.isOn("cegqi-stats") ){
     int tsize, itesize;
-    tsize = 0;itesize = 0;
-    d_sol->debugTermSize( d_orig_solution, tsize, itesize );
+    tsize = 0;
```

Comment:
This code is deleted on https://github.com/CVC4/CVC4/pull/5706.  We maybe should merge that PR first @HanielB .

Reply:
Or alternatively, we can delete this entire block on this PR, what do you think @abdoo8080 ?  You can delete the inferface for `debugTermSize` on your new module.

Reply:
I am fine with waiting for the other PR. Removed the block for now.

Reply:
Merged changes from PR.